src/libwalletqt/WalletManager.cpp

```diff
@@ -532,29 +537,26 @@ WalletManager::WalletManager(QObject *parent)
     , m_scheduler(this)
 {
     m_pimpl =  Monero::WalletManagerFactory::getWalletManager();
+    m_mutex_passphraseReceiver.lock();
```

Comment:
The mutex is not needed here. Initializing `m_passphraseReceiver(nullptr)` will work just fine